packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -48,6 +51,28 @@ typedef SetTextHandler = void Function(String text);
 /// Returned by [SemanticsConfiguration.getActionHandler].
 typedef SemanticsActionHandler = void Function(Object? args);
 
+/// The semantics merge rule generated from [SemanticsConfiguration.merger].
+class SemanticsMergeRule {
+  /// The child semantics configuration that are merging upward.
+  final List<SemanticsConfiguration> mergeUp = <SemanticsConfiguration>[];
+
+  /// The groups of child semantics configuration that want to merge together
+  /// and form a child [SemanticsNode].
+  ///
+  /// If the [SemanticsConfiguration] in the group is a semantics boundary or
```

Comment:
This is a list of semanticsCondfigurations. Which one is "the" one that determines this?